### ISSUE TITLE
Create a Pull request Template file.

### ISSUE THREAD

**Issue by BALaka-18** [BALaka-18/rake_new2#4]
## Description
Create a PULL_REQUEST_TEMPLATE.md file that must contain the skeleton of a PR description and generate the template each time a PR is created.

If you don't know how or what this template is, [read this link on creating Pull request templates](https://blog.axosoft.com/enhancing-pull-request-descriptions-templates/)

## Acceptance Criteria
- [ ] PULL_REQUEST_TEMPLATE.md file must be properly created.
- [ ] All instructions provided in the Description must be strictly followed.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Time Estimation
20 minutes

**Comment by sabarish98** [comment]
Hi @BALaka-18, Can I take this up and work on it? Since I am participating for HacktoberFest, I would be raising the PR in October

**Comment by BALaka-18** [comment]
> Hi @BALaka-18, Can I take this up and work on it? Since I am participating for HacktoberFest, I would be raising the PR in October

@sabarish98 yes absolutely, I am assigning you this issue.

**Comment by 16sweetyjain** [comment]
Hi @BALaka-18 , since this issue is open can I work on it?

**Comment by BALaka-18** [comment]
@sabarish98 make sure your PR comes in on 1st October positively before other PRs start pouring in, as they'll be needing the template.

**Comment by sabarish98** [comment]
Yes @BALaka-18, I would raise the PR by October 1 at the earliest.

**Comment by Ashuto7h** [comment]
This issue can be closed now

**Comment by sabarish98** [comment]
Closing this issue @BALaka-18
